content/200-concepts/100-components/250-preview-features/100-sql-server/001-sql-server-start-from-scratch.mdx

```diff
@@ -497,31 +497,31 @@ Prisma Client is an auto-generated, type-safe query builder based on your data m
 
 1. Run the code with the following command:
 
-    ```terminal
-    npx ts-node index.ts
-    ```
+   ```terminal
+   npx ts-node index.ts
+   ```
```

Comment:
Does this render correctly in the list now?

Reply:
<img width="545" alt="image" src="https://user-images.githubusercontent.com/4734297/124765456-6a136b00-df4f-11eb-8843-ea4eb38d660f.png">

Looks good